bumble/profiles/vcp.py

```diff
@@ -117,11 +118,12 @@ def __init__(
         )
 
         super().__init__(
-            [
+            characteristics=[
                 self.volume_state,
                 self.volume_control_point,
                 self.volume_flags,
-            ]
+            ],
+            included_services=included_services,
```

Comment:
Since we change on line#91, here we need
```suggestion
            included_services=list(included_services),
```